errors.ts

```diff
@@ -63,10 +65,10 @@ export function installUncaughtExceptionListener(): void {
 		console.log(callstack || err.toString());
 
 		try {
-		 	var analyticsService = $injector.resolve("analyticsService");
+			var analyticsService = $injector.resolve("analyticsService");
 			analyticsService.trackException(err, callstack);
 		} catch (e) {
-			$injector.resolve("$logger").error("Error while reporting exception: " + e);
+			console.log("Error while reporting exception: " + e);
```

Comment:
maybe a comment - "Do not replace with logger due to cyclic dependency and errors"